fitnessllm_dataplatform/utils/logging_utils.py

```
@@ -1,40 +1,131 @@
+    logging.getLogger("stravalib").setLevel(logging.WARNING)
+    logging.getLogger("google.auth._default").setLevel(logging.WARNING)
+
+    # Prevent adding duplicate handlers if this function is called multiple times
+    if logger.hasHandlers():
+        return logger
+
+    console_handler = logging.StreamHandler()
+    console_handler.setLevel(level)
+
+    formatter = logging.Formatter(
+        "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)